src/pytorch_lightning/strategies/deepspeed.py

```diff
@@ -331,7 +331,7 @@ def __init__(
         self.hysteresis = hysteresis
         self.min_loss_scale = min_loss_scale
 
-    def _load_config(self, config):
+    def _load_config(self, config: Optional[Union[_PATH, str, Dict[str, Any]]]) -> Optional[Dict[str, Any]]:
```

Comment:
```suggestion
    def _load_config(self, config: Optional[Union[_PATH, Dict[str, Any]]]) -> Optional[Dict[str, Any]]:
```

Reply:
also, mind updating the typing for config in init?

Reply:
Sure. Done!